<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="krishantha" name="DSS-Integration-Test" tests="8" failures="0" timestamp="18 Jan 2013 07:28:13 GMT" time="0.468" errors="0">
  <testcase name="@BeforeClass initialize" time="0.0" classname="org.wso2.carbon.dss.syntax.test.WhiteSpaceWithQueryParamsTest">
    <failure type="org.apache.axis2.AxisFault" message="Connection refused">
      <![CDATA[org.apache.axis2.AxisFault: Connection refused
	at org.apache.axis2.AxisFault.makeFault(AxisFault.java:430)
	at org.apache.axis2.transport.http.HTTPSender.sendViaPost(HTTPSender.java:197)
	at org.apache.axis2.transport.http.HTTPSender.send(HTTPSender.java:75)
	at org.apache.axis2.transport.http.CommonsHTTPTransportSender.writeMessageWithCommons(CommonsHTTPTransportSender.java:450)
	at org.apache.axis2.transport.http.CommonsHTTPTransportSender.invoke(CommonsHTTPTransportSender.java:277)
	at org.apache.axis2.engine.AxisEngine.send(AxisEngine.java:443)
	at org.apache.axis2.description.OutInAxisOperationClient.send(OutInAxisOperation.java:406)
	at org.apache.axis2.description.OutInAxisOperationClient.executeImpl(OutInAxisOperation.java:229)
	at org.apache.axis2.client.OperationClient.execute(OperationClient.java:165)
	at org.wso2.carbon.authenticator.stub.AuthenticationAdminStub.login(AuthenticationAdminStub.java:659)
	at org.wso2.carbon.automation.api.clients.authenticators.AuthenticatorClient.login(AuthenticatorClient.java:59)
	at org.wso2.carbon.automation.core.utils.environmentutils.EnvironmentBuilder.loginSetup(EnvironmentBuilder.java:448)
	at org.wso2.carbon.automation.core.utils.environmentutils.EnvironmentBuilder.dss(EnvironmentBuilder.java:210)
	at org.wso2.carbon.dss.DSSIntegrationTest.init(DSSIntegrationTest.java:53)
	at org.wso2.carbon.dss.syntax.test.WhiteSpaceWithQueryParamsTest.initialize(WhiteSpaceWithQueryParamsTest.java:57)
	at org.wso2.carbon.tests.dss.TestExecutorDss.execute(TestExecutorDss.java:52)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:60)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:106)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:111)
	at org.apache.maven.surefire.booter.ProviderFactory$ClassLoaderProxy.invoke(ProviderFactory.java:103)
	at $Proxy0.invoke(Unknown Source)
	at org.apache.maven.surefire.booter.SurefireStarter.invokeProvider(SurefireStarter.java:150)
	at org.apache.maven.surefire.booter.SurefireStarter.runSuitesInProcess(SurefireStarter.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:69)
Caused by: java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:351)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:213)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:200)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:559)
	at org.apache.commons.httpclient.protocol.ReflectionSocketFactory.createSocket(ReflectionSocketFactory.java:140)
	at org.apache.commons.httpclient.protocol.SSLProtocolSocketFactory.createSocket(SSLProtocolSocketFactory.java:130)
	at org.apache.commons.httpclient.HttpConnection.open(HttpConnection.java:707)
	at org.apache.commons.httpclient.MultiThreadedHttpConnectionManager$HttpConnectionAdapter.open(MultiThreadedHttpConnectionManager.java:1361)
	at org.apache.commons.httpclient.HttpMethodDirector.executeWithRetry(HttpMethodDirector.java:387)
	at org.apache.commons.httpclient.HttpMethodDirector.executeMethod(HttpMethodDirector.java:171)
	at org.apache.commons.httpclient.HttpClient.executeMethod(HttpClient.java:397)
	at org.apache.axis2.transport.http.AbstractHTTPSender.executeMethod(AbstractHTTPSender.java:621)
	at org.apache.axis2.transport.http.HTTPSender.sendViaPost(HTTPSender.java:193)
	... 62 more
... Removed 44 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeClass initialize -->
  <testcase name="@AfterClass testCleanup" time="0.0" classname="org.wso2.carbon.dss.syntax.test.WhiteSpaceWithQueryParamsTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at org.wso2.carbon.dss.syntax.test.WhiteSpaceWithQueryParamsTest.testCleanup(WhiteSpaceWithQueryParamsTest.java:104)
	at org.wso2.carbon.tests.dss.TestExecutorDss.execute(TestExecutorDss.java:52)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:60)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:106)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:111)
	at org.apache.maven.surefire.booter.ProviderFactory$ClassLoaderProxy.invoke(ProviderFactory.java:103)
	at $Proxy0.invoke(Unknown Source)
	at org.apache.maven.surefire.booter.SurefireStarter.invokeProvider(SurefireStarter.java:150)
	at org.apache.maven.surefire.booter.SurefireStarter.runSuitesInProcess(SurefireStarter.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:69)
... Removed 40 stack frames]]>
    </failure>
  </testcase> <!-- @AfterClass testCleanup -->
  <testcase name="@BeforeClass initialize" time="0.0" classname="org.wso2.carbon.dss.syntax.test.ReturnRequestStatusTest">
    <failure type="org.apache.axis2.AxisFault" message="Connection refused">
      <![CDATA[org.apache.axis2.AxisFault: Connection refused
	at org.apache.axis2.AxisFault.makeFault(AxisFault.java:430)
	at org.apache.axis2.transport.http.HTTPSender.sendViaPost(HTTPSender.java:197)
	at org.apache.axis2.transport.http.HTTPSender.send(HTTPSender.java:75)
	at org.apache.axis2.transport.http.CommonsHTTPTransportSender.writeMessageWithCommons(CommonsHTTPTransportSender.java:450)
	at org.apache.axis2.transport.http.CommonsHTTPTransportSender.invoke(CommonsHTTPTransportSender.java:277)
	at org.apache.axis2.engine.AxisEngine.send(AxisEngine.java:443)
	at org.apache.axis2.description.OutInAxisOperationClient.send(OutInAxisOperation.java:406)
	at org.apache.axis2.description.OutInAxisOperationClient.executeImpl(OutInAxisOperation.java:229)
	at org.apache.axis2.client.OperationClient.execute(OperationClient.java:165)
	at org.wso2.carbon.authenticator.stub.AuthenticationAdminStub.login(AuthenticationAdminStub.java:659)
	at org.wso2.carbon.automation.api.clients.authenticators.AuthenticatorClient.login(AuthenticatorClient.java:59)
	at org.wso2.carbon.automation.core.utils.environmentutils.EnvironmentBuilder.loginSetup(EnvironmentBuilder.java:448)
	at org.wso2.carbon.automation.core.utils.environmentutils.EnvironmentBuilder.dss(EnvironmentBuilder.java:210)
	at org.wso2.carbon.dss.DSSIntegrationTest.init(DSSIntegrationTest.java:53)
	at org.wso2.carbon.dss.syntax.test.ReturnRequestStatusTest.initialize(ReturnRequestStatusTest.java:58)
	at org.wso2.carbon.tests.dss.TestExecutorDss.execute(TestExecutorDss.java:52)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:60)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:106)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:111)
	at org.apache.maven.surefire.booter.ProviderFactory$ClassLoaderProxy.invoke(ProviderFactory.java:103)
	at $Proxy0.invoke(Unknown Source)
	at org.apache.maven.surefire.booter.SurefireStarter.invokeProvider(SurefireStarter.java:150)
	at org.apache.maven.surefire.booter.SurefireStarter.runSuitesInProcess(SurefireStarter.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:69)
Caused by: java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:351)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:213)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:200)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at com.sun.net.ssl.internal.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:559)
	at org.apache.commons.httpclient.protocol.ReflectionSocketFactory.createSocket(ReflectionSocketFactory.java:140)
	at org.apache.commons.httpclient.protocol.SSLProtocolSocketFactory.createSocket(SSLProtocolSocketFactory.java:130)
	at org.apache.commons.httpclient.HttpConnection.open(HttpConnection.java:707)
	at org.apache.commons.httpclient.MultiThreadedHttpConnectionManager$HttpConnectionAdapter.open(MultiThreadedHttpConnectionManager.java:1361)
	at org.apache.commons.httpclient.HttpMethodDirector.executeWithRetry(HttpMethodDirector.java:387)
	at org.apache.commons.httpclient.HttpMethodDirector.executeMethod(HttpMethodDirector.java:171)
	at org.apache.commons.httpclient.HttpClient.executeMethod(HttpClient.java:397)
	at org.apache.axis2.transport.http.AbstractHTTPSender.executeMethod(AbstractHTTPSender.java:621)
	at org.apache.axis2.transport.http.HTTPSender.sendViaPost(HTTPSender.java:193)
	... 62 more
... Removed 44 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeClass initialize -->
  <testcase name="@AfterClass testCleanup" time="0.0" classname="org.wso2.carbon.dss.syntax.test.ReturnRequestStatusTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at org.wso2.carbon.dss.syntax.test.ReturnRequestStatusTest.testCleanup(ReturnRequestStatusTest.java:146)
	at org.wso2.carbon.tests.dss.TestExecutorDss.execute(TestExecutorDss.java:52)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:60)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:106)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:111)
	at org.apache.maven.surefire.booter.ProviderFactory$ClassLoaderProxy.invoke(ProviderFactory.java:103)
	at $Proxy0.invoke(Unknown Source)
	at org.apache.maven.surefire.booter.SurefireStarter.invokeProvider(SurefireStarter.java:150)
	at org.apache.maven.surefire.booter.SurefireStarter.runSuitesInProcess(SurefireStarter.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:69)
... Removed 40 stack frames]]>
    </failure>
  </testcase> <!-- @AfterClass testCleanup -->
  <testcase name="testServiceDeployment" time="0.0020" classname="org.wso2.carbon.dss.syntax.test.WhiteSpaceWithQueryParamsTest">
    <skipped/>
  </testcase> <!-- testServiceDeployment -->
  <testcase name="testInsertOperation" time="0.0" classname="org.wso2.carbon.dss.syntax.test.WhiteSpaceWithQueryParamsTest">
    <skipped/>
  </testcase> <!-- testInsertOperation -->
  <testcase name="testSelectByNumber" time="0.0" classname="org.wso2.carbon.dss.syntax.test.WhiteSpaceWithQueryParamsTest">
    <skipped/>
  </testcase> <!-- testSelectByNumber -->
  <testcase name="deleteOperation" time="0.0" classname="org.wso2.carbon.dss.syntax.test.WhiteSpaceWithQueryParamsTest">
    <skipped/>
  </testcase> <!-- deleteOperation -->
  <testcase name="testServiceDeployment" time="0.0010" classname="org.wso2.carbon.dss.syntax.test.ReturnRequestStatusTest">
    <skipped/>
  </testcase> <!-- testServiceDeployment -->
  <testcase name="requestStatusNameSpaceQualifiedForInsertOperation" time="0.0" classname="org.wso2.carbon.dss.syntax.test.ReturnRequestStatusTest">
    <skipped/>
  </testcase> <!-- requestStatusNameSpaceQualifiedForInsertOperation -->
  <testcase name="requestStatusNameSpaceQualifiedForDeleteOperation" time="0.0" classname="org.wso2.carbon.dss.syntax.test.ReturnRequestStatusTest">
    <skipped/>
  </testcase> <!-- requestStatusNameSpaceQualifiedForDeleteOperation -->
  <testcase name="inOperationConcurrencyTest" time="0.0" classname="org.wso2.carbon.dss.syntax.test.ReturnRequestStatusTest">
    <skipped/>
  </testcase> <!-- inOperationConcurrencyTest -->
</testsuite> <!-- DSS-Integration-Test -->
